### PR TITLE
Tutorial improvements, reorganize

### LINKED ISSUE #23: move tutorial guide to book chapter
it's currently just a markdown file in the repository root, but not rendered on the website. 
https://github.com/uwhackweek/jupyterbook-template/blob/main/tutorial_guide.md 

on a related note, it would be good to turn the current 'example' notebook into a template notebook with required section headings
https://github.com/uwhackweek/jupyterbook-template/blob/main/book/tutorials/example/intro-ipyleaflet.ipynb

### PR DESCRIPTION
**Pull request by scottyhq**
- moves open science and open source software pages under the 'references' section.
- improve tutorial example notebook to highlight JupyterBook formatting features and "standard" format:
- add learning objectives, summary, references to "git, github, jupyterhub tutorial"
- fix remaining myst_substitution warnings
```
# Title
* Learning objectives
## Software
## Data
## Section
### Subsection
## Summary
## References
```

Closes #23

### CODE REVIEW

File: book/preliminary/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ We welcome all types of learners to our event, from professional software develo
 * navigate a [Jupyter Notebook](https://jupyter.org/) environment
 * conduct file management, text editing and other basic tasks from a command line interface
 * add and commit changes in Git, and push and pull content from GitHub
-* create simple scientific workflows in Python 
+* create simple scientific workflows in Python
```
Comment (jomey) [inline book/preliminary/index.md:8]: Style question - Should we capitalize the beginning of each bullet point?
For example:
```
* Create simple ...
```

File: book/tutorials/jupyter.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 # GitHub, Git, JupyterHub
 
-## [JupyterHub login setup](../preliminary/jupyterhub)
+```{admonition} Learning Objectives
+- use `git` to create copies of hackweek repositories on GitHub for modification
+- navigate the JupyterLab interface for writing and executing code
+```
 
-## [First time `git` setup](../preliminary/git)
+```{note}
```
Comment (jomey) [inline book/tutorials/jupyter.md:8]: Personally, I make this an {important} admonition

File: book/tutorials/jupyter.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
 # GitHub, Git, JupyterHub
 
-## [JupyterHub login setup](../preliminary/jupyterhub)
+```{admonition} Learning Objectives
+- use `git` to create copies of hackweek repositories on GitHub for modification
```
Comment (jomey) [inline book/tutorials/jupyter.md:4]: Also would capitalize the start for each list item

**Review by jomey — APPROVED** [review]

File: book/tutorials/example/intro-ipyleaflet.ipynb [hunk 4]
```diff
@@ -5,15 +5,30 @@
...
+    "# Example tutorial notebook\n",
     "\n",
-    "We will examine raster images from the [MODIS instrument](https://modis.gsfc.nasa.gov/data/). \"MODIS\" stands for \"MODerate Resolution SpectroRadiometer\". Moderate resolution refers to the fact that MODIS data has at best a 250 meter pixel posting, but single images cover hundreds of kilometers and with two sensors currently in orbit, we currently get [daily views of the entire globe](https://worldview.earthdata.nasa.gov/)!"
+    "```{admonition} Learning Objectives\n",
+    "**This is an example tutorial notebook for UW hackweeks**\n",
+    "- understand basic Jupyter Book formatting of .ipynb files\n",
+    "- use an open source Python library [ipyleaflet](https://github.com/jupyter-widgets/ipyleaflet) to create interactive maps\n",
+    "```\n",
+    "☝️ This formatting is a Jupyter Book [admonition](https://jupyterbook.org/content/content-blocks.html#notes-warnings-and-other-admonitions)"
+   ]
```
Comment (jomey) [inline book/tutorials/example/intro-ipyleaflet.ipynb:16]: Want to add another sentence and add the link to the multitude of options for Jupyter Books:
https://jupyterbook.org/content/index.html

File: book/preliminary/index.md (hunk 1 above)
Comment (scottyhq) [reply book/preliminary/index.md:8]: Probably... Although not sure how to enforce it other than the manual review

**Pull request merged**